src/specifications/profile-resolution/profile-resolution-specml.xml

```diff
@@ -3,1772 +3,1370 @@
-        show any groups or structures in the form of elements not described here. Depending on the
-        merge directive, all groups will be given either by the catalog source(s)
-          (<src>merge/as-is</src>) or the profile (<src>merge/custom</src>), or there will be no
-        groups at all (no merge directive).</p>
-    </section>
-    <section id="target-back-matter">
-      <head>Back matter</head>
-      <p>The back matter in target catalogs is assembled from the back matter of their source
-        profiles in combination with the back matter of source catalogs. In both catalogs and
-        profiles, <src>back-matter</src> is comprised of (multiple uses of either)
-          <src>citation</src> or <src>resource</src>. Either of these elements may be
-        cross-referenced from any location in their source documents by means of a URI fragment
-        identifier on an <src>href</src> flag, for example <code>&lt;resource id="document-X</code>>
-        may be referenced from inside its containing document by <code>&lt;link
-          href="#document-X"></code> appearing elsewhere. Whether an element inside back matter must
-        be included in the target, depends on whether such a cross-reference has also been included
-        in the same target.</p>
-      <p>Specifically, any <src>citation</src> or <src>resource</src> inside <src>back-matter</src>
-        either in the profile, or in any imported catalog, is considered eligible for propagation
-        into the target if either of two conditions obtains:</p>
-      <ul>
-        <li>
-          <p>The element (<src>citation</src> or <src>resource</src>) is referenced by any element
-            inside the profile metadata.</p>
-        </li>
-        <li>
-          <p>The element is referenced from a link or anchor (<src>link</src> or <src>a</src>
-            element) inside controls, parameters, or any contents (for example, paragraphs inside
-              <tgt>group/part</tgt>) included in the resolved profile. The link takes the form of a
-            URI fragment identifier <code>#id</code>, where <code>id</code> is the value of the
-              <src>@id</src> attribute of the <src>citation</src> or <src>resource</src>.</p>
-        </li>
-      </ul>
-      <p>The assembly of the target <tgt>back-matter</tgt> occurs by including copies of all
-        eligible <src>citation</src> elements followed by all eligible <src>resource</src> elements,
-        in the same order as given in their sources.</p>
-      <p>Additionally, combination rules (see next section) will affect whether and how duplicate or
-        competing <src>citation</src> or <src>resource</src> elements are handled, to remove or
-        combine multiple occurrences of the same referenced object.</p>
+
+
+    <section id="readingthis">
+      <head>Reading This Document</head>
+      <section id="terminology">
```

Comment:
It might make sense to point to: https://pages.nist.gov/OSCAL/concepts/terminology/ for many of these in addition to the model pages.